kernel/src/common/math.rs

```diff
@@ -8,14 +8,7 @@ use core::intrinsics as int;
 //      https://doc.rust-lang.org/core/intrinsics/
 //  add additional wrappers as needed
 
-pub fn powf32(base: f32, exponent: f32) -> f32 {
-    unsafe { int::powf32(base, exponent) }
-}
-
-pub fn powif32(base: f32, exponent: i32) -> f32 {
-    unsafe { int::powif32(base, exponent) }
-}
-
+/// Provide `sqrtf32` with the unsafe hidden.
```

Comment:
Are `powf32` and `powif32` unused? Doesn't seem that bad to keep possibly useful math functions around.

Reply:
I'm not excited about having unsafe functions exposed without a use case. I'm already a little weary of the use case we have in capsules (tmp006).